Jenkinsfile_CNP

```diff
@@ -100,9 +100,9 @@ withPipeline(type, product, component) {
        if (env.BRANCH_NAME.startsWith("PR") || env.BRANCH_NAME == 'master') {
           environmentOfDependencies = "aat"
        }
-       env.CCD_DATA_STORE_API_BASE_URL = "http://ccd-data-store-api-${environmentOfDependencies}.service.core-compute-${environmentOfDependencies}.internal"
+       env.CCD_DATA_STORE_API_BASE_URL = "http://ccd-data-store-api-${environmentOfDependencies}.platform.hmcts.net"
```

Comment:
add ${environmentOfDependencies}. before platform.hmcts.net

Reply:
already there

Reply:
it should be repeated as the first one is part of the host name